scripts/check_nightly.py

```diff
@@ -1,60 +1,62 @@
 #!/usr/bin/env python3
 
-import glob
+import os
 import pathlib
 import re
 import sys
+import typing
 
 import nayduck
 
-expensive_pattern = '#[cfg(feature = "expensive_tests")]'
-test_pattern = '#[test]'
+IGNORED_SUBDIRS = ('target', 'target_expensive', 'sandbox')
```

Comment:
can you comment on what this whole script does?

Reply:
Done.